Anlab.Jobs.SendMail/Anlab.Jobs.SendMail.csproj

```diff
@@ -1,20 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk;Microsoft.NET.Sdk.Publish">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <AssemblyVersion>1.0.0.8</AssemblyVersion>

```

Comment:
@sprucely Bump up the AssemblyVersion for the jobs.

Reply:
resolved with [edec7b3](https://github.com/ucdavis/Anlab/pull/919/commits/edec7b3a8e3820ac7b03c591990c8e9152e926f6)